src/include/symbol_table/symbole_table.h

```diff
@@ -0,0 +1,101 @@
+#ifndef JUCC_SYMBOL_TABLE_SYMBOL_TABLE_H
+#define JUCC_SYMBOL_TABLE_SYMBOL_TABLE_H
+
+#include <iostream>
+#include <map>
```

Comment:
is iostream necessary?
use unordered_map

Reply:
Iostream is not necessary can use unordered_map 👍